### ISSUE TITLE
Problem with detecting 

### ISSUE THREAD

**Issue by macsmy** [amdegroot/ssd.pytorch#227]
Hello, 
I am passing through this course: COMPUTER VISION A-Z™. One of thesis is running pre-train SSD on a test video. In the tutorial, everything is detected as it should be - in my case, using the same code, model and video, I can detect something olny on few frames. Is it a common problem? What should I improve?

**Comment by zacario-li** [comment]
@macsmy which dataset label do you use?

**Comment by macsmy** [comment]
@zacario-li i am using pre-train model ssd300_mAP_77.43_v2.pth

**Comment by zacario-li** [comment]
@macsmy so you use voc label, and voc label only contains 20 classes, list as below:  
**VOC_CLASSES** = (  # always index 0
    'aeroplane', 'bicycle', 'bird', 'boat',
    'bottle', 'bus', 'car', 'cat', 'chair',
    'cow', 'diningtable', 'dog', 'horse',
    'motorbike', 'person', 'pottedplant',
    'sheep', 'sofa', 'train', 'tvmonitor')  
So I think your test video file maybe don't have many objects above.

**Comment by macsmy** [comment]
@zacario-li i know that, the video include dog and a person. As i said in the first post, in the tutorial, where they are using exactly the same model and video, the results are great. when i am running same project, it can detect person only on just a few frames.

**Comment by zacario-li** [comment]
@macsmy I don't know what happened in your code. I run the code, and it works well. After I fine-tuned with my datasets, it still works well. And I use PyTorch v0.4.

**Comment by krxat** [comment]
@zacario-li  Can you please explain how you fine-tuned it to your dataset?
I am trying to train the network and it is not working. Please help.
Thanks
